platform/darwin/src/MGLStyleLayer.h.ejs

```diff
@@ -6,7 +6,7 @@
   const enumProperties = locals.enumProperties;
 -%>
 // This file is generated.
-// Edit platform/darwin/scripts/generate-style-code.js, then run `make style-code-darwin`.
+// Edit platform/darwin/scripts/generate-style-code.js, then run `make style-code`.
```

Comment:
Good catch! The make file invocation was changed previously but this was not updated to match.

However, I think we should update this comment to use the renamed [darwin-style-code](https://github.com/mapbox/mapbox-gl-native/blob/release-ios-v3.5.0-android-v5.0.0/Makefile#L268) instead of `style-code` since the latter runs codegen for core and all platforms.